pkg/cmd/config.go

```diff
@@ -328,6 +328,9 @@ func createAndPrintManifest(printOptions *printOpts) error {
 	case "equinixmetal":
 		cfg.Set(yamled.Path{"cloudProvider", "equinixmetal"}, providerVal)
 		cfg.Set(yamled.Path{"cloudProvider", "external"}, true)
+	case "vmware-cloud-director":
+		cfg.Set(yamled.Path{"cloudProvider", "vmware-cloud-director"}, providerVal)
```

Comment:
Same here -- shouldn't we use the JSON tag (`vmwareCloudDirector`) here?

Reply:
Fixed this one

Reply:
I don't see that this is fixed, it still shows the same value for me.

Reply:
Done.